src/Main.java

```diff
@@ -2,60 +2,52 @@ public class Main {
     public static void main(String[] args) {
 
         //task1
-        System.out.println("");
         char clientOS = 0;
         if (clientOS == 0) {
             System.out.println("Установите версию приложения для iOS по ссылке");
-        } else {
+        } else if (clientOS == 1) {
             System.out.println("Установите версию приложения для Android по ссылке");
         }
 
         //task2
-        System.out.println("");
         int clientDeviceYear = 2015;
-        if (clientDeviceYear >= 2015) {
-            if (clientOS == 0) {
+        if ((clientDeviceYear >= 2015) && (clientOS == 0)) {
```

Comment:
```suggestion
        if (clientDeviceYear >= 2015 && clientOS == 0) {
```
не нужно писать лишние скобки